lib/gui/app/modules/image-writer.js

```diff
@@ -127,9 +156,16 @@ exports.performWrite = (image, drives, onProgress) => {
     })
 
     const flashResults = {}
+    const analyticsData = {
```

Comment:
Rather than creating the `analyticsData` object twice, would it make sense for `exports.flash` to pass its `analyticsData` object into  `exports.performWrite` ?